src/sysrepo.c

```diff
@@ -4110,6 +4110,7 @@ sr_apply_changes(sr_session_ctx_t *session, uint32_t timeout_ms)
         sr_errinfo_merge(&err_info, cb_err_info);
         sr_errinfo_new(&err_info, SR_ERR_CALLBACK_FAILED, "User callback failed.");
     }
+    SR_LOG_DBG("%s for %s %s.", __func__, sr_ds2str(session->ds), err_info ? "failed" : "success");
```

Comment:
Just to keep it unified and not use any function names, please make the message something akin to
```
Applying "<DS>" datastore changes failed/suceeded.
```